.github/jitx-client/scripts/run-ocdb-tests.sh

```diff
@@ -9,13 +9,25 @@ jitx check-install
 echo "/root/.jitx/user.params:"
 cat /root/.jitx/user.params
 
+#==============================================
+#==== public pcb-* macro evaluation tests =====
+#==============================================
+
+# Those macro evaluation tests take into account the root stanza.proj as it is included in the generated test-evaluate/stanza.proj
 echo "Searching for pcb objects and generating tests..."
 python3.8 scripts/evaluate_pcb_objects.py
 cd test-evaluate/
 echo "Launching pcb object tests..."
 jitx run-test test/evaluate/api
 cd ..
 
+#==============================================
+#====== Unit-tests and integration-tests ======
+#==============================================
+
+# Those tests need to take into account the root stanza.proj so that they find the source files from the ocdb repo and not a pkg from the jitx-client docker image
+export STANZA_CONFIG=/app/open-components-database/
```

Comment:
Is there a .stanza file here? You can pass `stanza.proj` files as args to `jitx` as well.

Reply:
Oh no there is none, this is wrong